packages/gen-typescript-declarations/src/test/gen-ts_goog_test.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * @license
+ * Copyright (c) 2017 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt The complete set of authors may be found
+ * at http://polymer.github.io/AUTHORS.txt The complete set of contributors may
+ * be found at http://polymer.github.io/CONTRIBUTORS.txt Code distributed by
+ * Google as part of the polymer project is also subject to an additional IP
+ * rights grant found at http://polymer.github.io/PATENTS.txt
+ */
+
+import {assert} from 'chai';
+import * as fs from 'fs';
+import * as glob from 'glob';
+import * as path from 'path';
+
+import {Config, generateDeclarations} from '../gen-ts';
+
+const fixtures = path.join(__dirname, '..', '..', 'src', 'test', 'fixtures');
+const goldens = path.join(__dirname, '..', '..', 'src', 'test', 'goldens_goog');
+
+suite('generateDeclarations (goog)', () => {
```

Comment:
this is nearly all duplicated, why not just update the main test to run the test twice with two different configs?

Reply:
The choice was deliberate, and guided the maxims: "If you do it twice, copy-paste, if you do it three times, abstract" and "test code should optimize for legibility even more than most".

That said, I'm not dead set on it